src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard movie={movie} key={movie.imdbId} />)}
```

Comment:
it's hard to read code that's written in one line, let's fix it a bit
```suggestion
    {movies.map(movie => (
    	<MovieCard movie={movie} key={movie.imdbId} />
    ))}
```